js/filter-control.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var $ = require('jquery');
+
+function FilterControl($selector) {
+  this.$body = $selector;
```

Comment:
Not a big deal, but I keep seeing `$body` and thinking the entire `body` element. I know `$body` is the convention, but I usually use the name `$element` for this kind of stuff. Maybe we can change this going forward?